actions/transaction_brokernodes.go

```diff
@@ -74,3 +83,47 @@ func (usr *TransactionBrokernodeResource) Create(c buffalo.Context) error {
 
 	return c.Render(200, r.JSON(res))
 }
+
+func (usr *TransactionBrokernodeResource) Update(c buffalo.Context) error {
+	req := transactionUpdateReq{}
+	parseReqBody(c.Request(), &req)
+
+	// Get transaction
+	t := &models.Transaction{}
+	transactionError := models.DB.Eager("DataMap").Find(t, c.Param("id"))
+
+	trytes := giota.Trytes(req.Trytes)
+	iotaTransaction, iotaError := giota.NewTransaction(trytes)
+
+	if transactionError != nil || iotaError != nil {
+		return c.Render(400, r.JSON(map[string]string{"error": "No transaction found"}))
+	}
+
+	address, addError := giota.ToAddress(t.DataMap.Address)
+	validAddress := addError == nil && address == iotaTransaction.Address
+	validMessage := giota.Trytes(t.DataMap.Message) == iotaTransaction.SignatureMessageFragment
```

Comment:
This message one might fail sometimes because iota will pad the end of the message with a bunch of 9s if we don't fill it all the way up.

Reply:
Could do something like this:  https://github.com/oysterprotocol/brokernode/blob/master/services/iota_wrappers.go#L405